application/common/config/main.php

```diff
@@ -205,8 +205,14 @@
         'passwordGracePeriodExtension'  => '+7 days',
         'inviteLifespan'                => Env::get('INVITE_LIFESPAN', '+1 month'),
         'inviteGracePeriod'             => Env::get('INVITE_GRACE_PERIOD', '+3 months'),
-        'inactiveUserDeletionEnable'    => Env::get('INACTIVE_USER_DELETION_ENABLE', false),
-        'inactiveUserPeriod'            => Env::get('INACTIVE_USER_PERIOD', '+18 months'),
+        'inactiveUser'                  => [
```

Comment:
You could use `Env::getArrayFromPrefix` if you either rename `DEACTIVATE_INSTRUCTIONS_URL` or move it outside the `inactiveUser` block. (Of course, the latter would require a change to `Emailer` as well.)  There are examples of this near the top of `main.php`. One benefit is the code saved here, but there are other potential benefits in the Terraform code if all related variables have a common prefix.

Reply:
After our Slack conversation, I decided it's not worth making this change. As you pointed out, two existing variables would break backward compatibility. Let's keep it as it is.